src/data_importing/readme.md

```diff
@@ -102,3 +104,39 @@ This mapping indicates to Deep Lynx all it needs to know in order to transform t
 Data is processed by the system automatically. As long as an ingested piece of data is mapped to a type the system will automatically take the data and type mapping - using them to transform and load the data into the Deep Lynx system. This processing is handled by a mix of database triggers, scheduled jobs, and a long polling process handled by the Deep Lynx system itself. See the diagram above for reference.
 
 
+# BYPASS
+
+Deep Lynx _does_ provide the ability to bypass this data storage methodology. This can be useful if you already know the Metatype your data corresponds to and that your data already maps to the Metatype keys. Bypassing the methodology might be beneficial as your data will be instantly queryable and not held back, waiting for type mapping.
```

Comment:
I think in line with our discussion earlier, it might be good to modify the wording here. Calling out that this data flow is intended to be used for specific purposes including: 
1. populating test or demo data when in early stages of development or for demonstration purposes and 2. providing an API for modifying an existing data set (creation of data which must be done manually and providing a means to update and delete nodes, their properties, and edges)

Either here or above where the reader is directed to the BYPASS section it might also be beneficial to explicitly call out the intended process for data ingestion to deep lynx and the reasons why. Also would another name besides "BYPASS" convey the intended use more clearly?

Reply:
Good idea. I'll get this changed.

Reply:
Corrected. Used what you said and called it Method 1 and Method 2 (Not Recommended) - that should go a long way in discouraging or at least indicating why they shouldn't use it.